java/src/org/openqa/selenium/grid/config/Config.java

```diff
@@ -59,14 +61,22 @@ default <X> X getClass(String section, String option, Class<X> typeOfClass, Stri
     }
   }
 
+  String DELIM_KEY = "\u001E";
+  String DELIMITER = DELIM_KEY + "=\"record-separator\"";
```

Comment:
How do you use this in a toml file?

Reply:
Delimiters aren't specified in client requests; these are inserted by the API. Clients have the ability to define lists of maps, and the API flattens these into lists of key/value strings. The delimiters enable the API to determine where each group of associated key/value strings ends.
If clients ever call the `getAll()` method directly, they'll need to be aware of these break points as well, but I strongly doubt that anyone will ever want/need to do so.

Reply:
However, TOML actually supports Unicode character constants in the same format that Java and JSON use.

Reply:
Maybe I can understand better through an example. How would a TOML file look like with this delimiter?

Reply:
```
[example]
"\u001E" = "record-separator"
```

Reply:
Perhaps I should mark these DELIMITER constants as **INTERNAL**. Do we have an established pattern for marking and documenting such constants?

Reply:
I still do not get it. Apologies. 

Can you share a real TOML file you have used to test this? I need help to understand how the delimiter looks like.

Reply:
The delimiter will not be used in TOML files. It's only used internally by the Config API to indicate the end of an associated group of settings. This eliminates reliance in the `display-name` setting for this purpose and the need to sort the settings list.

Reply:
I used an ASCII control code as the key name so it's highly unlikely that anyone will ever add a new setting that will be mistaken for this delimiter. The current strategy of using `display-name` as the delimiter requires that the list of key/value strings returned by `getAll()` must be sorted and is at risk for breakage if someone adds a new setting whose sort position precedes `display-name` (e.g. - `debug-port`).  
Delimiters are added by `Config.toEntryList()` and removed by `NodeOptions.addDriverConfigs()`. Their sole purpose is to mark the end of each group of driver configuration settings. No one will ever explicitly include these delimiters in their driver configuration files.

Reply:
@diemol: At this point, the only references to the delimiter (other than the methods mentioned above) are in the unit tests for the `getAll()` method.